tests/services/test_scheduler.py

```
@@ -164,6 +164,57 @@ def test_sets_permissions_error_on_cold_location_when_not_writable(self, db_sess
+
+    def test_caution_low_disk_space_still_reports_real_write_permission_error(
+        self, db_session, tmp_path, monkeypatch
+    ):
+        """Only critical disk pressure suppresses write-permission classification."""
+        from app.services.scheduler import check_storage_permissions_job_func
+
+        cold_path = tmp_path / "cold"
+        cold_path.mkdir()
+        loc = self._make_cold_location(db_session, str(cold_path))
+
+        monkeypatch.setattr("app.services.scheduler.os.access", lambda path, mode: mode != 2)
```

Ruff (0.15.11)

[warning] Unused lambda argument: `path`

(ARG005)